packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/download-rclone/download-rclone.routing.js

```diff
@@ -3,62 +3,55 @@ import {
   RCLONE_GUIDE,
   DOWNLOAD_RCLONE_FILENAME,
   DOWNLOAD_RCLONE_FILETYPE,
-  REGION_CAPACITY,
-  S3_REGION_CAPACITY,
-} from '../../../../../../components/users/download-rclone/download-rclone.constants';
+  COLD_ARCHIVE_DEFAULT_REGION,
+  COLD_ARCHIVE_STATES,
+} from './download-rclone.constants';
 
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state(
-    'pci.projects.project.storages.cold-archive.users.download-rclone',
-    {
-      url: '/rclone/download?userId',
-      views: {
-        modal: {
-          component: 'pciProjectUsersDownloadRclone',
-        },
+  $stateProvider.state(`${COLD_ARCHIVE_STATES.S3_USERS_DOWNLOAD_RCLONE}`, {
+    url: '/rclone/download?userId',
+    views: {
+      modal: {
+        component: 'pciProjectsProjectStoragesColdArchiveUsersDownloadRclone',
       },
-      layout: 'modal',
-      resolve: {
-        breadcrumb: () => null, // Hide breadcrumb
-        file: /* @ngInject */ () => {
-          return {
-            fileName: DOWNLOAD_RCLONE_FILENAME,
-            fileType: DOWNLOAD_RCLONE_FILETYPE,
-          };
-        },
-        userId: /* @ngInject */ ($transition$) => $transition$.params().userId,
-        user: /* @ngInject */ (
-          PciStoragesColdArchiveService,
-          projectId,
-          userId,
-        ) => PciStoragesColdArchiveService.get(projectId, userId),
-        regions: /* @ngInject */ (PciProject, projectId) =>
-          PciProject.getStorageRegions(
-            projectId,
-            REGION_CAPACITY,
-          ).then((regions) => regions.map(({ name }) => name)),
-        storageS3Regions: /* @ngInject */ (PciProject, projectId) =>
-          PciProject.getS3StorageRegions(
-            projectId,
-            S3_REGION_CAPACITY,
-          ).then((regions) => regions.map(({ name }) => name)),
-        rcloneGuide: /* @ngInject */ (coreConfig) => {
-          return get(RCLONE_GUIDE, coreConfig.getUser().ovhSubsidiary);
-        },
-        goBack: /* @ngInject */ (goToUsers) => goToUsers,
-        downloadRCloneConfig: /* @ngInject */ (
-          PciStoragesColdArchiveService,
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: () => null, // Hide breadcrumb
+      file: /* @ngInject */ () => {
+        return {
+          fileName: DOWNLOAD_RCLONE_FILENAME,
+          fileType: DOWNLOAD_RCLONE_FILETYPE,
+        };
+      },
+      userId: /* @ngInject */ ($transition$) => $transition$.params().userId,
+      user: /* @ngInject */ (
+        PciStoragesColdArchiveService,
+        projectId,
+        userId,
+      ) => PciStoragesColdArchiveService.get(projectId, userId),
+      regions: /* @ngInject */ () => {
+        return [COLD_ARCHIVE_DEFAULT_REGION];
+      },
+      storageS3Regions: /* @ngInject */ () => {
+        return [COLD_ARCHIVE_DEFAULT_REGION];
+      },
+      rcloneGuide: /* @ngInject */ (coreConfig) => {
+        return get(RCLONE_GUIDE, coreConfig.getUser().ovhSubsidiary);
```

Comment:
```suggestion
        return RCLONE_GUIDE[coreConfig.getUser().ovhSubsidiary] || RCLONE_GUIDE.DEFAULT;
```
You can add the default value if there is a missing subsidiary.

Reply:
completed